### ISSUE TITLE
Website Analytics Dashboard

### ISSUE THREAD

**Issue by pawarspeaks** [pawarspeaks/TechNexus#57]
**We look forward to a feature with which we can analyse the total visits, interactions on the TechNexus platform...which will help us better improvise on how we can improve and track the website's performance.**

**Comment by kaustubhdeokar** [comment]
@pawarspeaks  can i work on this ?

**Comment by TecHAyusH6476** [comment]
@pawarspeaks Please assign this to me

**Comment by pawarspeaks** [comment]
@kaustubhdeokar @TecHAyusH6476 how about working together on this one!
Thank you so much for your enthusiasm, I would like to suggest that both of you can get connected over a communication platform, plan out things for this (cause this will be an important very important feature for tracking user interactions) may be try out how we can develop an independent dashboard or something with only access so it should not be publicly accessible unless shared..

Thanks again, Happy Open Sourcing!🚀
please let me know if you guys need any more contributors to work with you.

**Comment by TecHAyusH6476** [comment]
@pawarspeaks Thanks for assigning 
I would like to go through the data and page once 
Will let u know if needed any info

**Comment by pawarspeaks** [comment]
> @pawarspeaks Thanks for assigning 
> I would like to go through the data and page once 
> Will let u know if needed any info

Sure please collaborate with @kaustubhdeokar

**Comment by TecHAyusH6476** [comment]
Yeah will do so

**Comment by kaustubhdeokar** [comment]
@TecHAyusH6476 Hey man how would you like to collaborate ? discord ? dropped you a ping in discord

**Comment by TecHAyusH6476** [comment]
@kaustubhdeokar hey lets connect over discord

**Comment by kaustubhdeokar** [comment]
great,
![Screenshot from 2024-10-12 21-46-00](https://github.com/user-attachments/assets/bb1c9a35-e902-4705-963f-c26806597441)

**Comment by kaustubhdeokar** [comment]
Hi, @pawarspeaks  me and ayush were seeing the repo & site, we do not see any backend code nor any requests/responses. Is the repo somewhere else ? or we need a backend repo ?

**Comment by pawarspeaks** [comment]
Hi @kaustubhdeokar this project was created initially to ease out contributing for beginner so currently we don't have one...
A few things we can do...
1] create a completely new repo in Dev Code Organization https://github.com/Dev-Code-Community
 (so that we can keep the front end and backend independent without making things complex for beginner contributors and then host the backend over render)
2] Or else we can just use this one itself and create a new directory for backend
Let mw know your thoughts..

If creating a new repo is needed I'll initialize a new repo in https://github.com/Dev-Code-Community
and update you

**Comment by kaustubhdeokar** [comment]
better to have separate repos , so in future deployment and scaling can be separate .. but we aren't serving anything as of now.. we need to change that, create and persist all events, which until now are stored in json format ..

**Comment by kaustubhdeokar** [comment]
@pawarspeaks let me know if you wanna have a quick chat over discord/g meet .. cc: @TecHAyusH6476

**Comment by TecHAyusH6476** [comment]
Yeah lets decide it first to avoid any rework and freeze the initial phase

**Comment by kaustubhdeokar** [comment]
@pawarspeaks  @TecHAyusH6476  is today 7pm a good time ?

**Comment by pawarspeaks** [comment]
@kaustubhdeokar @TecHAyusH6476 apologies for the delay in responding..yeah today 7pm works lets hop on the voice channels of the Dev Code Community discord server.
Invite to the server: https://discord.gg/7fnjneZVq8

see you guys!

**Comment by kaustubhdeokar** [comment]
great! waiting in the meeting room voice channel

**Comment by TecHAyusH6476** [comment]
Sorry I was out of station couldnt check

**Comment by pawarspeaks** [comment]
Hey @TecHAyusH6476 no worries.please join the conversation at https://discord.gg/G2acRSSdhX and connect with @kaustubhdeokar
